rpc/handlers_servicer.go

```diff
@@ -0,0 +1,26 @@
+package rpc
+
+import (
+	"net/http"
+
+	"github.com/labstack/echo/v4"
+)
+
+func (s *rpcServer) PostV1ServicerRelay(ctx echo.Context) error {
+
+	// TODO: Move to the servicer module
+	servicerModuleName := "servicer"
+	found := false
+	for _, am := range s.GetBus().GetUtilityModule().GetActorModules() {
```

Comment:
NIT: I think you can simply this using `slices.IndexFunc` https://stackoverflow.com/a/38654444/768439